CONTRIBUTING

```diff
@@ -0,0 +1,102 @@
+# How to contribute
+
+Welcome to `digits` library for GoLang.

```

Comment:
Remove welcome statement.  Welcome statement is more for the `README` file than this file.